prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -250,11 +251,40 @@ class CompareCsvFiles : CliktCommand(
     }
 }
 
+class LivdTableDownload() : CliktCommand(
+    name = "livd-table",
```

Comment:
Maybe change to livd-table-download?  Not sure.  Could make it more obvious.

Reply:
Done. thanks